src/ui/components/PrefixBadge.jsx

```diff
@@ -135,32 +90,16 @@ function PrefixBadgeButton({ breakpoint, theme, setBreakpointPrefixBadge }) {
     return null;
   }
 
-  console.log({ showPrefixBadge });
-
   const prefixBadge = breakpoint.options.prefixBadge;
   const isColor = isColorPrefix(prefixBadge);
```

Comment:
Pretty sure we can delete lines 93 and 94, and then we can delete `isColorPrefix`... and then we can delete `PREFIX_COLORS`